### PR TITLE
Divyanshu | Test | Support of bulk entry creation

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cvhkyec

### CODE REVIEW

File: apps/web-giddh/src/app/services/ledger.service.ts [hunk 1]
```diff
@@ -122,6 +122,32 @@ export class LedgerService {
...
 
+    /**
+     * Create Bulk Ledger from bank transaction
+     *
+     * @param {BlankLedgerVM} model
+     * @param {string} accountUniqueName
+     * @returns {Observable<BaseResponse<LedgerResponse[], BlankLedgerVM>>}
+     * @memberof LedgerService
+     */
+    public CreateBulkLedger(model: BlankLedgerVM, accountUniqueName: string): Observable<BaseResponse<LedgerResponse[], BlankLedgerVM>> {
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/services/ledger.service.ts]: format in camel case

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/services/ledger.service.ts (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/services/ledger.service.ts]: Done

**Review by dilpreetsinghofficial — APPROVED** [review]

File: apps/web-giddh/src/app/ledger/ledger.component.ts [hunk 2]
```diff
@@ -951,6 +961,43 @@ export class LedgerComponent implements OnInit, OnDestroy {
...
+                let filteredCredit: any[] = [];
+
+                this.lc.bankTransactionsDebitData.forEach(transaction => {
+                    if (transaction.transactions[0].selectedAccount?.name) {
+                        filteredDebit.push(transaction);
+                    }
+                });
+                this.lc.bankTransactionsCreditData.forEach(transaction => {
+                    if (transaction.transactions[0].selectedAccount?.name) {
+                        filteredCredit.push(transaction);
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/app/ledger/ledger.component.ts]: don't create multiple array. Push in single array only

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/ledger/ledger.component.ts (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/ledger/ledger.component.ts]: DOne

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**